docs/accessibility/accessibility-module-feature-overview.md

```diff
@@ -25,34 +30,59 @@ If the chart enables drilldown or is otherwise updating its data dynamically, Hi
 
 The Accessibility module is compatible with most screen readers in modern browsers, including JAWS, NVDA, Narrator, VoiceOver, and TalkBack.
 
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/8Sew47xr-EU" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
 Low vision features
 -------------------
+With the Accessibility module, Highcharts comes with built in support for Windows High Contrast Mode. Charts can also be viewed in full-screen for additional separation of elements. [Pattern fills](https://www.highcharts.com/docs/accessibility/patterns-and-contrast) are supported, as well as [premade themes](https://www.highcharts.com/docs/chart-design-and-style/themes) with higher contrast. Series markers also have different shapes to help distinguish between data points without use of color. Due to the responsive nature of Highcharts, screen zooming tools are well supported.
+
+Demo with higher contrast themes: 
+<iframe style="width: 100%; height: 470px; border: none;" src='https://www.highcharts.com/samples/embed/highcharts/accessibility/accessible-switch-theme' allow="fullscreen"></iframe>
 
-With the accessibility module, Highcharts comes with built in support for Windows High Contrast Mode. Charts can also be viewed in full screen for additional separation of elements. [Pattern fills](https://www.highcharts.com/docs/accessibility/patterns-and-contrast) are supported, as well as [premade themes](https://www.highcharts.com/docs/chart-design-and-style/themes) with higher contrast. Series markers also have different shapes to help distinguish between data points without use of color. Due to the responsive nature of Highcharts, screen zooming tools are well supported.
+[View demo code](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/accessibility/accessible-switch-theme)
```

Comment:
Note for the demos task: HTML in this demo an error reported in the JSFiddle.
![image](https://github.com/user-attachments/assets/c6d9300d-6373-42cd-ac93-f7a1ec6f2a1d)

Reply:
BTW: this is fixed on another PR, so resolving the comment.